docs/KBS_URI.md

```diff
@@ -0,0 +1,33 @@
+# KBS URI
+
+## Introduction
+
+To uniquely identify every resource/key in the CoCo Key Broker System, a __KBS URI__ is defined.
```

Comment:
Why don't we call such URI `Resource URI`? If we use the name `KBS URI` to refer to a URI such as "kbs://", it will be easy to confuse with the "https://" URI of the KBS instance.

Reply:
Agree with @jialez0. KBS URI sounds like it points to a KBS not a resource.

Reply:
Great, nice suggestion. Thanks!